src/ALogOutputter.cpp

```diff
@@ -18,30 +18,6 @@ ALogOutputter::ALogOutputter() { }
  */
 ALogOutputter::~ALogOutputter() { }
```

Comment:
I think this definition and its declaration in the header can also be removed.

Reply:
ALogOutPutter is an abstract base class. The destructor is virtual, so that inheriting classes can call their own destructor. Since I declare the destructor I also need to implement it.

So this one is needed